contracts/MemberTeller.sol

```diff
@@ -49,7 +49,7 @@ contract MemberTeller {
         }
         if (bytes4(data) == ENCODED_SIG_REMOVE_OWNER && safe == to) {
             // Ensure data is at minimum, the length required for the below logic.
-            require(data.length >= 44, "incorrect data length");
+            require(data.length == 100, "incorrect data length");
```

Comment:
how did we get these numbers?

Reply:
They were provided to us by the Quantstamp audit.